From 56e0cf1c66b921e0a8abb844a4de8a0322e15311 Mon Sep 17 00:00:00 2001
From: Martin Gieseking <martin.gieseking@uos.de>
Date: Fri, 26 Jul 2019 15:27:20 +0200
Subject: [PATCH] use std::round to round color value (fixes #116)

---
 src/Color.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/Color.cpp b/src/Color.cpp
index 949fcba5..cff8e982 100644
--- a/src/Color.cpp
+++ b/src/Color.cpp
@@ -40,7 +40,7 @@ const Color Color::TRANSPARENT(uint32_t(0xff000000));
 
 static inline uint8_t double_to_byte (double v) {
 	v = max(0.0, min(1.0, v));
-	return uint8_t(floor(255*v+0.5));
+	return uint8_t(round(255.0*v));
 }
 
 
